liberty-maven-plugin/src/main/java/io/openliberty/tools/maven/applications/DeployMojoSupport.java

```diff
@@ -114,7 +114,14 @@ protected void installLooseConfigWar(MavenProject proj, LooseConfigData config)
             config.setProjectRoot(proj.getBasedir().getAbsolutePath());
             config.setSourceOnDiskName("${"+PROJECT_ROOT_NAME+"}");
             // Set the property to include the variable definition in liberty-plugin-variable-config.xml
-            proj.getProperties().setProperty("liberty.var."+PROJECT_ROOT_NAME, proj.getBasedir().getAbsolutePath());
+            String projectPath = proj.getBasedir().getAbsolutePath();
+            proj.getProperties().setProperty("liberty.var."+PROJECT_ROOT_NAME, projectPath);
+            if (copyLibsDirectory == null) { // in container mode, copy dependencies from .m2 dir to the target dir to mount in container
+                copyLibsDirectory = new File(proj.getBasedir(), PROJECT_ROOT_TARGET_LIBS);
+            } else if (!copyLibsDirectory.getAbsolutePath().startsWith(projectPath)) {
```

Comment:
The only issue here is if the basedir or copyLibsDirectory have things such as ".." or "." in their absolute paths, they may not match properly.  Instead of `.getAbsolutePath()`, you could use `getCanonicalPath()`.

Reply:
Ok, I think this is important when we go to compare the paths. When we are just using them as in the existing code we can stick with `.getAbsolutePath()`.